### PR TITLE
Lg/add quota section Moved to top-level

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

## Description of changes being made
Add a section about quota management and move it under a new top-level Multi-tenancy section.

https://jira.mesosphere.com/browse/DCOS-56371
## Checklist
- [x] Make sure you change all affected versions - 1.14 feature
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/1.14/multi-tenancy/resource-mgmt-primitives/index.md [hunk 1]
```diff
@@ -0,0 +1,395 @@
+---
+layout: layout.pug
+navigationTitle: Resource Management Primitives
+title: Resource Management Primitives
+menuWeight: 10
+excerpt: Reserving resources to support multi-tenancy
+render: mustache
+model: /1.14/data.yml
+---
+Resources in DC/OS can be reserved and prioritized using a combination of roles, reservations, quotas, and weights. These features are provided by Apache Mesos, at the core of DC/OS and are referred to as `Primitives`, as they are only accessible via the API and have not yet been integrated into the DC/OS UI or CLI. A user requires good monitoring in place of available/used resources when working with quotas, reservations, and weights.
```
Comment (vinodkone) [inline pages/1.14/multi-tenancy/resource-mgmt-primitives/index.md]: minor comment. 
"as they are only accessible" --> "as most of them are only accessible"

File: pages/1.14/multi-tenancy/resource-mgmt-primitives/index.md [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+
+Resource management in this context refers to concepts such as reservations of resources on agents, resource quotas, and weights (priorities) for frameworks. These are useful for a number of scenarios, such as configuring multi-tenant environments, where multiple teams or projects co-exist on the same DC/OS cluster, and the available resources (CPU, RAM, disk, and ports) must be carved up and guaranteed for each cluster with guaranteed quotas. Secondly, with mixed workloads on a single cluster where one class of frameworks may have a higher weight (priority) than another, resources with high priority should be able to deploy faster than a lower weight framework.
+
+This page covers the multi-tenancy primitives: Multi-Tenant quota management primitives, two examples of real-world scenarios, implementation instructions, and reference links.
+
+# Multi-Tenant Quota Management Primitives
```
Comment (vinodkone) [inline pages/1.14/multi-tenancy/resource-mgmt-primitives/index.md]: Multi-Tenant Resource Management Primitives

File: pages/1.14/multi-tenancy/quota-management/index.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+dcos quota get dev
+```
+
+You can also view quota information in the DC/OS UI by going to the Quota tab in the **Services** view.
+
+# Updating quota
+For updating existing quota on a group, use the following command:
+
+```bash
+dcos quota update dev --cpu 20 --me, 2048
```
Comment (vinodkone) [inline pages/1.14/multi-tenancy/quota-management/index.md]: "--me," --> "--mem"

File: pages/1.14/quota-management/index.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
+---
```
Comment (vinodkone) [inline pages/1.14/quota-management/index.md]: this page should be deleted!

File: pages/1.14/quota-management/prepare-to-use-quota/index.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
+layout: layout.pug
```
Comment (vinodkone) [inline pages/1.14/quota-management/prepare-to-use-quota/index.md]: this page should be deleted!

File: pages/1.14/quota-management/working-with-quotas/index.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
+---
```
Comment (vinodkone) [inline pages/1.14/quota-management/working-with-quotas/index.md]: this page should be deleted.

**Comment by lsgunnlsgunn** [comment]
changes made.

**Comment by ck4adventure** [comment]
This PR will be merged under https://github.com/mesosphere/dcos-docs-site/pull/2489

**Pull request closed without merging**